channel/tests/ChannelAPITest.m

```diff
@@ -30,7 +30,8 @@ function createChannel(testCase)
 				testCase.Logger = MonsterLog(Config);
 				
 				Config.setupNetworkLayout(testCase.Logger);
-				Stations = setupStations(Config, testCase.Logger);
+				Sites = setupSites(Config, testCase.Logger);
+				Stations = [Sites.Cells];
```

Comment:
The keyword stations should be removed and replaced with cells.

Reply:
Resolved in latest commit across project